<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogMessageType</class>
 <widget class="QDialog" name="DialogMessageType">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <widget class="QComboBox" name="cbRulesetMode">
     <item>
      <property name="text">
       <string>All rules must apply (AND)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>At least one rule must apply (OR)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>No rule must apply (NOR)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0" rowspan="3" colspan="2">
    <widget class="QTableView" name="tblViewRuleset">
     <property name="showGrid">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QToolButton" name="btnRemoveRule">
     <property name="toolTip">
      <string>Remove ruleset</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset theme="list-remove">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QRadioButton" name="rbAssignManually">
     <property name="text">
      <string>Assi&amp;gn manually</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QRadioButton" name="rbAssignAutomatically">
     <property name="text">
      <string>Assign a&amp;utomatically</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>145</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QToolButton" name="btnAddRule">
     <property name="toolTip">
      <string>Add ruleset</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset theme="list-add">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
